build/lib/locFunc.js

```diff
@@ -356,20 +344,33 @@ function renameVscodeLangpacks() {
         }
         let locADSFolder = path.join('.', 'i18n', `ads-language-pack-${langId}`);
         let locVSCODEFolder = path.join('.', 'i18n', `vscode-language-pack-${langId}`);
+        let translationDataFolder = path.join(locVSCODEFolder, 'translations');
+        let location = path.join('.', 'resources', 'xlf');
```

Comment:
Would suggest a better name than `location` since that's a bit generic. `xlfFolder` seems better based on the above ones